crates/rolldown/src/dev/bundling_task.rs

```diff
@@ -47,7 +46,7 @@ impl BundlingTask {
       Ok(()) => {}
       Err(_) => {
         let mut build_status = self.dev_data.status.lock().await;
-        build_status.is_building = false;
+        build_status.try_to_idle().expect("FIXME: Should not unwrap here");
```

Comment:
This part is also called if `build_status.try_to_idle()?` in `build_inner` returns an error.

Reply:
good catch. Something's wrong here.

Reply:
This one is to make sure if errors happen, the build status could be reset to idel.